reportportal_client/service.py

```diff
@@ -139,6 +139,17 @@ def finish_launch(self, end_time, status=None):
         logger.debug("finish_launch - Stack: %s", self.stack)
         return _get_msg(r)
 
+    def stop_launch(self, end_time, status=None):
+        data = {
+            "end_time": end_time,
+            "status": status
+        }
+        url = uri_join(self.base_url, "launch", self.launch_id, "stop")
+        r = self.session.put(url=url, json=data)
+        self.stack.pop()
```

Comment:
Are you sure about the work with the stack while closing the whole execution process? I don't think that this is somehow useful after using this method because no any work made after that. 

Thus, having such logging below is not good too.